plugins/formal-verification/src/org/jetbrains/kotlin/formver/conversion/SpecialMethods.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2010-2023 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.formver.conversion
+
+import org.jetbrains.kotlin.formver.scala.silicon.ast.*
+import org.jetbrains.kotlin.formver.scala.silicon.ast.Exp.*
+import org.jetbrains.kotlin.formver.scala.toScalaBigInt
+
+object SpecialMethods {
+    private val thisArg = LocalVar(ThisVariableName, Type.Ref)
```

Comment:
Is there a benefit to using a new special kind of name here?  I think you could just use an anonymous name or a local name.